hack/tools.mk

```
@@ -80,6 +81,8 @@ SKAFFOLD_VERSION ?= v2.13.2
 YQ_VERSION ?= v4.44.6
 # renovate: datasource=github-releases depName=ironcore-dev/vgopath
 VGOPATH_VERSION ?= v0.1.7
+# renovate: datasource=github-releases depName=crate-ci/typos
+TYPOS_VERSION ?= v1.29.0
```

Minor detail, we could directly go to `v1.29.4` to avoid directly receiving a Renovate PR after merging this one
```suggestion
TYPOS_VERSION ?= v1.29.4`
```

---

fixed